packages/discord.js/src/util/Util.ts

```diff
@@ -316,7 +321,7 @@ class Util extends null {
    * @returns {Object}
    * @private
    */
-  static cloneObject(obj) {
+  static cloneObject(obj: object) {
```

Comment:
```suggestion
  static cloneObject<T>(obj: T)T {
```

why not something like this?